### ISSUE TITLE
Control animateIn

### ISSUE THREAD

**Issue by gmzbri** [vasturiano/globe.gl#142]
Hi, 
Is it possible to affect on the initial _animateIn_ to remove the scale and speed?

Thanks for the great work

**Comment by vasturiano** [comment]
No, it's only possible to toggle the build-in animation on/off via the `animateIn` config option.

But you can always do any camera changes you'd like programmatically via the [pointOfView](https://github.com/vasturiano/globe.gl#render-control), so you can build your own custom animation that way.
